docs/arvo/network.md

```diff
@@ -15,9 +15,9 @@ All we've written up until now are just shell commands that produce a
 value and then disappear. To listen for and receive messages from other
 planets, we'll need an app. Let's look at a very simple one:
 
-    ::  There is no love that is not an echo
+    ::  Accepts any noun from dojo and prints it out
     ::
-    ::::  /hoon/echo/examples/app
```

Comment:
Got it. Will revert. Please note that it may not be apparent to the naive user, so do we want to mention this convention somewhere?